### ISSUE TITLE
NavButton redesign

### ISSUE THREAD

**Issue by firstChairCoder** [olatunjiemanuel/RickAndMortyApp#1]
The huge floating buttons does not seem like good UI, imo. You could try a bottom tab with overflow: "hidden" that hides other things from showing or something else.
It's up to you, chief.

**Comment by olatunjiemanuel** [comment]
yes, honestly I thought so too. I wanted to make sure the navigation and pagination was working before I redesign it. ill work on it today. I also want to add a feature where the current page is going to be displayed between the navButtons

**Comment by olatunjiemanuel** [comment]
Proposed new navButton redesign
<img width="148" alt="Screenshot 2022-01-10 at 13 15 50" src="https://user-images.githubusercontent.com/93155336/148765026-59772525-ca1d-4f07-8b8f-8f60b1aee4d6.png">
